src/topology/algebra/monoid.lean

```diff
@@ -274,6 +275,58 @@ continuous.comp (continuous_pow n) h
 
 end has_continuous_mul
 
+section op
+
+open opposite
+
+/-- Put the same topological space structure on the opposite monoid as on the original space. -/
+instance [_i : topological_space α] : topological_space αᵒᵖ :=
+topological_space.induced (unop : αᵒᵖ → α) _i
+
+variables [topological_space α] [monoid α]
+
+lemma continuous_unop : continuous (unop : αᵒᵖ → α) := continuous_induced_dom
+lemma continuous_op : continuous (op : α → αᵒᵖ) := continuous_induced_rng continuous_id
+
+variables [has_continuous_mul α]
+
+/-- If multiplication is continuous in the monoid `α`, then it also is in the monoid `αᵒᵖ`. -/
+instance : has_continuous_mul αᵒᵖ :=
```

Comment:
Can you please get `@[to_additive]` working for the lemmas in this file and `topology.algebra.group`?   I was lazy when I wrote it :)

Reply:
I already tried when I first saw it but I think this is a harder task than it looks because the file `algebra.opposite` was not written with `@[to_additive]` and I believe we should refactor that file before getting this to work...

Reply:
Ah, good point!

Reply:
Should we write about his on Zulip or in the issues?

Reply:
Sure, good idea.